### PR TITLE
Add UpdateAttributes for Rawls

### PR DESCRIPTION
**Pull request by marctalbott**
**PR checklist**
- [x] For each library you've modified here, decide whether it requires a major, minor, or no version bump. (Click [here](/broadinstitute/workbench-libs/blob/develop/CONTRIBUTING.md) for guidance)

If you're doing a **major** or **minor** version bump to any libraries:
- [x] Bump the version in `project/Settings.scala` `createVersion()`
- [x] Update `CHANGELOG.md` for those libraries
- [x] I promise I used `@deprecated` instead of deleting code where possible

In all cases:
- [x] Replace the appropriate version hashes in `README.md` and the `CHANGELOG.md` for any libs you changed with `TRAVIS-REPLACE-ME` to auto-update the version string
- [x] Get two thumbsworth of PR review
- [x] Verify all tests go green (CI _and_ coverage tests)
- [x] Squash commits and **merge to develop**
- [x] Delete branch after merge

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -70,6 +70,8 @@ object Dependencies {
 
   val fs2Io: ModuleID = "co.fs2" %% "fs2-io" % "1.0.4"
 
+  val rawlsModel: ModuleID = "org.broadinstitute.dsde" %% "rawls-model" % "0.1-0d02c8ce-SNAP" exclude("com.typesafe.scala-logging", "scala-logging_2.11") exclude("com.typesafe.akka", "akka-stream_2.11")
```
Comment (aweng98) [inline project/Dependencies.scala:73]: don't forget to change `0.1-0d02c8ce-SNAP` before merge.

File: serviceTest/CHANGELOG.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ SBT dependency: `"org.broadinstitute.dsde.workbench" %% "workbench-service-test"
 ### Changed
 - automation tests now request the `cloud-billing` scope instead of the `cloud-platform` scope in the `AuthTokenScopes.billingScopes`
 variable. This better reflects end-user behavior.
-- add `updateAcl` endpoint to Rawls.workspaces
+- add `updateAcl` and `updateAttributes` endpoint to Rawls.workspaces
```
Comment (aweng98) [inline serviceTest/CHANGELOG.md:17]: 👍admittedly, I don't update this part every time.

File: serviceTest/src/test/scala/org/broadinstitute/dsde/workbench/service/Rawls.scala [hunk 3]
```diff
@@ -194,6 +196,16 @@ trait Rawls extends RestClient with LazyLogging {
       logger.info(s"Updating acl for workspace $name in $namespace")
       patchRequest(url + s"api/workspaces/$namespace/$name/acl?inviteUsersNotFound=$inviteUsersNotFound", aclUpdates.map(e => e.toMap))
     }
+
+    def updateAttributes(namespace: String, name: String, attributeUpdates: List[AttributeUpdateOperation])(implicit token: AuthToken): Unit = {
```
Comment (aweng98) [inline serviceTest/src/test/scala/org/broadinstitute/dsde/workbench/service/Rawls.scala]: `patchRequest` returns a `String`, this method would be little more useful if also returns `String`

File: project/Dependencies.scala (hunk 1 above)
Comment (marctalbott) [reply project/Dependencies.scala:73]: This is actually the same hash that is used by Orchestration. I'm not sure why it's a SNAP, but I'm inclined to leave it as is for consistency's sake. https://github.com/broadinstitute/firecloud-orchestration/blob/112fdbc0dca8e115f4fdfa262e13e83ccb926219/project/Dependencies.scala#L25

**Review by aweng98 — APPROVED** [review]

**Review by ansingh7115 — APPROVED** [review]

File: serviceTest/src/test/scala/org/broadinstitute/dsde/workbench/config/CommonConfig.scala [hunk 4]
```diff
@@ -10,7 +11,7 @@ trait CommonConfig {
     protected val usersConfig = config.getConfig("users")
 
     private val notSoSecretPassword = usersConfig.getString("notSoSecretPassword")
-    private val userDataJson = JSON.parseFull(scala.io.Source.fromFile(usersConfig.getString("userDataPath")).getLines.mkString).get.asInstanceOf[Map[String, Map[String,String]]]
+    private val userDataJson = scala.io.Source.fromFile(usersConfig.getString("userDataPath")).getLines().mkString.parseJson.convertTo[Map[String, Map[String, String]]]
```
Comment (marctalbott) [inline serviceTest/src/test/scala/org/broadinstitute/dsde/workbench/config/CommonConfig.scala:14]: After importing the `rawlsModel`, sbt was not happy with using `JSON.parseFull` which has been deprecated, so I had to change how it was getting parsed to use a non-deprecated approach.

**Pull request merged**